clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1176,10 +1176,11 @@ public static String toHtml() {
         for (ApiKeys key : ApiKeys.values()) {
             // Key
             b.append("<h5>");
+            b.append("<a name=\"The_Messages_" + key.name + "\">");
```

Comment:
FYI, I tested the use of `id` instead of `name` and it seems to work as well. So if you prefer I can update the PR to use the `id` attribute.